packages/glsl_analyzer/package.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name: glsl_analyzer
+description: |
+  Language server for GLSL (autocomplete, goto-definition, formatter, and more)
+homepage: https://github.com/nolanderc/glsl_analyzer
+licenses:
+  - GPL-3.0
+languages:
+  - OpenGL
+categories:
+  - LSP
+
+source:
+  id: pkg:github/nolanderc/glsl_analyzer@v1.4.4
+  asset:
+    - target: darwin_x64
+      file: x86_64-macos.zip
+      bin: bin/glsl_analyzer
+    - target: darwin_arm64
+      file: aarch64-macos.zip
+      bin: bin/glsl_analyzer
+    - target: linux_x64
+      file: x86_64-linux-musl.zip
+      bin: bin/glsl_analyzer
+    - target: linux_arm64
+      file: aarch64-linux-musl.zip
+      bin: bin/glsl_analyzer
+    - target: win_x64
+      file: x86_64-windows.zip
+      bin: bin/glsl_analyzer.exe
+    - target: win_arm64
+      file: aarch64-windows.zip
+      bin: bin/glsl_analyzer.exe
+
+schemas:
+  lsp: vscode:https://raw.githubusercontent.com/nolanderc/glsl_analyzer-vscode/main/package.json
+
```

Comment:
This schema currently only contains one configuration item which is only used by the VSCode client extension: `glsl-analyzer.path` (sets the path to the executable). I think it'd mostly add noise and confusion so let's exclude the schema for now:

```suggestion
```